### PR TITLE
fix heapster rc file jinja syntax error

### PR DESCRIPTION
**Pull request by freehan**
ref: #12765

### CODE REVIEW

File: cluster/addons/cluster-monitoring/google/heapster-controller.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% set heapster_memory = "300Mi" -%}
-{% if pillar['num_nodes'] is defined and pillar['num_nodes'] is not None -%}
+{% if pillar['num_nodes'] is defined and pillar['num_nodes'] >= 0 -%}
```
Comment (zmerlynn) [inline cluster/addons/cluster-monitoring/google/heapster-controller.yaml]: I would rather these be something like:
```
{% set num_nodes = pillar.get('num_nodes', -1) %}
{% if num_nodes >= 0 %}
  {% set heapster_memory = (200 + num_nodes * 12)|string + "Mi" -%}
{% endif -%}
```

**Comment by zmerlynn** [comment]
@freehan: Thanks!

**Comment by brendandburns** [comment]
self-merging since this is borked @ HEAD

**Comment by brendandburns** [comment]
(and thanks for fixing my bug...)

**Pull request merged**